components/pages/feature-cards.tsx

```diff
@@ -4,28 +4,48 @@ import { featureCards } from "@/config/contents"
 import { Card, CardDescription, CardTitle } from "@/components/ui/card"
 import { Icons } from "@/components/icons"
 
+
+
 export default function FeatureCards() {
   return (
-    <section className="bg-slate-50 dark:bg-slate-900">
-      <div className="container space-y-8 py-12 text-center lg:py-20">
+    <section style={{ backgroundColor: '#f8fafc' }}>
+      <div style={{
+        maxWidth: '1200px',
+        margin: '0 auto',
+        padding: '3rem 1rem',
+        textAlign: 'center'
+      }}>
         {featureCards.header || featureCards.subheader ? (
           <HeadingText subtext={featureCards.subheader}>
             {featureCards.header}
           </HeadingText>
         ) : null}
-        <div className="grid grid-cols-1 gap-4 md:grid-cols-3">
+        <div style={{
+          display: 'grid',
+          gridTemplateColumns: 'repeat(auto-fit, minmax(300px, 1fr))',
+          gap: '1rem'
+        }}>
           {featureCards.content.map((cards) => {
             const Icon = Icons[cards.icon || "blank"]
 
             return (
               <Card
```

Comment:
We should actually be using the Card file generated in the `CustomCards` repo instead of importing this. All cards should be generated in that manner